<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="computation_namespace" default="vehicleA"/>
  <arg name="rx_com_topic" default="/rx_com"/>

  <node pkg="computation_model" type="compute_model" name="compute_model" ns="$(arg computation_namespace)" output="screen">
    <param name="batch_size" value='100' />
    <param name="t_op" value='0.01' />
    <param name="app_flo" value='1000' />

    <remap from="/computation/status" to="/$(arg computation_namespace)/computation/status"/>
    <remap from="/computation/cooperation_status" to="/$(arg computation_namespace)/computation/cooperation_status"/>
    <remap from="/computation/updated_lane_waypoints_array" to="/$(arg computation_namespace)/computation/updated_lane_waypoints_array"/>
    <remap from="/lane_waypoints_array" to="/$(arg computation_namespace)/lane_waypoints_array"/>
    <remap from="/rx_com" to="/$(arg rx_com_topic)"/>

  </node>

</launch>
